deployment/liquidityMining/deployLiquidityMiningV2.js

```diff
@@ -64,7 +64,7 @@ task('deployLiquidityMiningV2', 'deploy liquidity mining V2 contracts')
       // FOR TESTING LOCALLY
       // fairLaunchConfigs[i].rewardTokens = [rewardToken.address];
       // END
-      if (fairLaunchConfigs[i].address != undefined) {
+      if (typeof(fairLaunchConfigs[i].address != 'undefined')) {
```

Comment:
why changes needed here?

Reply:
i return it back to fairLaunchConfigs[i].address != undefined, nothing update here.